frontend/src/components/ReferralCreationModal/ReferralCreationModal.tsx

```diff
@@ -23,6 +24,17 @@ export default function ReferralCreationModal(props: ReferralCreationModalProps
   const { jobPost, closeModal, ...modalProps } = props
   const [ activeStep, setActiveStep ] = useState(0)
 
+  // Data needed from the user
+  /* eslint-disable */
+  const [firstName, setFirstName] = useState<string>('')
+  const [lastName, setLastName] = useState<string>('')
+  const [email, setEmail] = useState<string>('')
+  const [phone, setPhone] = useState<string>('')
+  const [recommendation, setRecommendation] = useState<string>('')
+  const [resumeFilePath, setresumeFilePath] = useState<string>('')
+  const [employeeId, setEmployeeId] = useState<string>('')
```

Comment:
This will be part of that object, as `id` (just happens to be how i did it. i thought it would be easier for you, but hadn't read this yet haha)